src/main/java/io/fusionauth/domain/SecureIdentity.java

```diff
@@ -27,8 +27,6 @@
  * @author Daniel DeGroff
  */
 public class SecureIdentity {
-  
```

Comment:
Our TODO remover in client builder is not perfect, but now the remnants are gone